tools/osx/notarize.py

```diff
@@ -12,13 +14,15 @@
 import time
 from typing import List, Pattern, Tuple
 
+import requests
+
 BUNDLE_IDENTIFIER = os.getenv("BUNDLE_IDENTIFIER", "org.nuxeo.drive")
-NOTARIZATION_USERNAME = os.getenv("NOTARIZATION_USERNAME", "")
-NOTARIZATION_PASSWORD = os.getenv("NOTARIZATION_PASSWORD", "")
+NOTARIZATION_USERNAME = os.environ["NOTARIZATION_USERNAME"]
```

Comment:
This is expected to prefer `os.environ` to fail early when those envars are not set.